frontend/src/components/dashboard/create/govsg/GovsgMessages.tsx

```diff
@@ -77,12 +89,17 @@ export const GovsgMessages = ({ campaignId }: GovsgMessagesProps) => {
   }
 
   useEffect(() => {
-    void fetchGovsgMessages(selectedPage)
+    void fetchGovsgMessages(search, selectedPage)
```

Comment:
Sorry, I didn't catch this previously but can I ask why we need this `useEffect`?

When the user changes the page, the change is already handled in `handlePageChange` (which refetches data). Is there still a need to trigger `fetchGovsgMessages` again?

Reply:
Without this, the first render will be empty (nobody clicked a page yet).

But, I have removed the `selectedPage` dependency now so that we don't run fetch twice per page change.

Thanks!